robotbase/src/main/java/frc/robot/commands/EnableSpeakerLockCommand.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+import frc.robot.state.RobotState.DriveControlState;
+
+public class EnableSpeakerLockCommand extends Command {
+  @Override
+  public void initialize() {
+    Robot.state.setDriveControlState(DriveControlState.SPEAKER_LOCK);
+    Robot.shooterLimelight.setPipeline(4);
```

Comment:
We'll need to add an enum, or better way to switch pipelines other than hardcoding a number.